litex/build/lattice/diamond.py

```diff
@@ -57,6 +57,7 @@ def _build_files(device, sources, vincpaths, build_name):
     tcl.append("prj_run Map -impl implementation")
     tcl.append("prj_run PAR -impl implementation")
     tcl.append("prj_run Export -impl implementation -task Bitgen")
+    tcl.append("prj_run Export -impl implementation -task Jedecgen")
```

Comment:
This does not work for me (Diamond 3.9 on Linux x86_64):

    Wrong # arg -task
    Jedecgen is not a valid task name.
        while executing
    "prj_run Export -impl implementation -task Jedecgen"
        (file "top.tcl" line 10)

Do we actually need to generate JEDEC files? I would think these don't even apply to modern FPGAs anymore... Any particular reason you want this present?